byebyefirst/README.md

```diff
@@ -0,0 +1,35 @@
+## byebyefirst
+
+### Instructions
+
+Write a function that takes a slice of strings and returns a new slice without the first element. If the slice is empty, return the empty slice.
+
+### Expected function
+
+```go
+func ByeByeFirst(strings []string) []string {
+}
+```
+
+### Usage
+
+Here is a possible program to test your function :
+
+```go
+package main
+
+func main() {
+	fmt.Println(ByeByeFirst([]string{}))
```

Comment:
Just add this example and its good to go:
	
```fmt.Println(ByeByeFirst([]string{"one arg"}))```

Reply:
added